N3.js/src/main/scala/org/w3/banana/n3js/model.scala

```diff
@@ -0,0 +1,7 @@
+package org.w3.banana
+package n3js
+package model
```

Comment:
Interesting remark. I found a bit overkill to create one directory with 2 small files in it.

Would that style be ok instead?

```
package org.w3.banana
package n3js

package model {

  final class MGraph[S, P, O](var graph: plantain.model.Graph[S, P, O])

  final case class BNode(label: String)

}
```